docs/source/markdown/podman-pod-create.1.md.in

```diff
@@ -143,6 +143,11 @@ but only by the pod itself.
 
 @@option replace
 
+@@option restart
+
+The restart policy is set at the pod level so all containers in the pod have the same restart policy configured.
+When the restart policy is set to **no** or **never**, the exit-policy of the pod is set to **stop**. This will ensure that infra container also exits once all the containers in the pod exit.
```

Comment:
Why only on `no` or `never`?  What about pods with not infra container?  Is this information important to readers?

Reply:
On `no` and `never`, the containers will never restart. Since exit-policy is used to stop the pod completely when the containers exit, I thought to use it for this use case as the containers will never be restarted and we can make the pod exit once they exit. I may have misunderstood the purpose of `--exit-policy`.

Reply:
@vrothberg does this make sense or should `exit-policy` not be changed based on the pod's `restart policy`?

Reply:
Can we break this out into another issue/card?  Once we have the `--restart` behavior nailed down, I hope it would be easier to put the puzzle together.

Reply:
Removed it and will add in another PR